packages/api/functions/express/routes/program.js

```diff
@@ -0,0 +1,62 @@
+import express from 'express'
+import wrapAsync from '../wrap-async'
+import {
+  createProgram,
+  getProgram,
+  updateProgram,
+  scanPrograms,
+} from '../../../controllers/program'
+import { getProjects } from '../../../controllers/project'
+
+const programRouter = express.Router()
+
+programRouter.post('/', wrapAsync(async (req, res) => {
+  const { program } = req.body
+  const userId = req.cognitoUser.id
+  const programItem = await createProgram({ userId, program })
+
+  res.json({ program: programItem })
+}))
+
+programRouter.put('/:programId', wrapAsync(async (req, res) => {
+  const { programId } = req.params
+  const { program } = req.body
+  const programItem = await updateProgram({ programId, program })
+
+  res.json({ program: programItem })
+}))
+
+programRouter.get('/', wrapAsync(async (req, res) => {
+  const programs = await scanPrograms()
+  res.json({ programs })
+}))
+
+programRouter.get('/:programId', wrapAsync(async (req, res) => {
```

Comment:
ah good old express :)

Should we change API Gateway configs to add each of the routes?

Reply:
Not necessary. There's not really any reason to do this unless those routes have some special API Gateway configuration you need to apply.